centreon/www/include/eventLogs/xml/data.php

```diff
@@ -46,10 +46,16 @@
 require_once __DIR__ . '/../../../../bootstrap.php';
 require_once __DIR__ . '/../Paginator.php';
 require_once __DIR__ . '/PaginationRenderer.php';
+require_once __DIR__ . '/../../../class/centreonDB.class.php';
+require_once __DIR__ . '/../../../class/centreonLog.class.php';
+require_once __DIR__ . '/../../../class/exceptions/CentreonDbException.php';
 
 // Connect to DB
-$pearDB = $dependencyInjector['configuration_db'];
-$pearDBO = $dependencyInjector['realtime_db'];
+$pearDB = new CentreonDB(CentreonDB::LABEL_DB_CONFIGURATION);
+$pearDBO = new CentreonDB(CentreonDB::LABEL_DB_REALTIME);
```

Comment:
Each time you create a new instance of this class, you create a new database connection.
To save on the number of open connections, it's preferable to use an existing connection, and therefore to use those already available in the $dependencyInjector variable.

Reply:
done